dev_setup.sh

```diff
@@ -287,6 +287,7 @@ function install_venv() {
     # about this version.  Update whenever a new version is released and
     # verified functional.
     curl https://bootstrap.pypa.io/3.3/get-pip.py | "${VIRTUALENV_ROOT}/bin/python" - 'pip==18.0.0'
+    ls "${VIRTUALENV_ROOT}/bin/pip" > /dev/null 2>&1
```

Comment:
Would it be clearer to use the bash file flag?:
```bash
[ -f "$VIRTUALENV_ROOT/bin/pip" ]
```

Reply:
Indeed it would. Didn't realize that could be used outside of if statements. Good to know. will fix

Reply:
Update has been pushed.